### DISCUSSION TITLE
New Feature Request, Voice Prompts

### DISCUSSION THREAD

**Discussion by Sonwon1** [osmandapp/OsmAnd#17821] [Ideas]
We need a selection to setup the voice prompts so the roads and cities are always pronounced in the native tongue of the country you are navigating in. For example I am an English speaker in Germany. I want the direction commands in English however when the German street names and cities are pronounced in English they are pronounced completely wrong. Voice prompts in German are good for streets and cities however then the directions are not easy to understand.

**Comment by mikehgentry** [comment]
I'm not 100% sure that's possible using Text To Speech for all of it, because you're relying on the phone's selected TTS engine (and I think most or all of them are monolingual).

In theory you could set up a hybrid system that used pre-recorded snippets for the directions but TTS for the place names etc. But it'd be awkward to use (you'd have to set the TTS to the 'wrong' language for the entire phone) so I'm guessing it won't be implemented any time soon.

**Reply by Sonwon1** [reply]
Thank you for the reply!

Maybe if I listen to the German long enough I will learn the directions. :)

**Reply by sjvudp** [reply]
[mikehgentry](https://github.com/mikehgentry): "I think most or all of them are monolingual":
Actually I have an Android app (text2speech, TTS util) that can speak one text in several languages (which is quite funny).
Of course it requires that the corresponding languages be installed ;-)

**Reply by Sonwon1** [reply]
So there is hope for me! :)
